styles/global.css

```diff
@@ -109,6 +127,14 @@ footer {
 @media (max-width: 768px) {
     .footer-container {
         flex-direction: column; 
+    
+    }
+    .header-b{
```

Comment:
Use logical and descriptive class names that represent the purpose of the element